include/z64item.h

```diff
@@ -90,6 +90,7 @@ typedef enum {
     /* 0x5C */ GI_POTION_GREEN,
     /* 0x5D */ GI_POTION_BLUE,
     /* 0x5E */ GI_FAIRY,
+    /* 0x60 */ GI_MILK = 0x60,
```

Comment:
```suggestion
    /* 0x60 */ GI_MILK_BOTTLE = 0x60,
```
Really nice work addressing all the comments. Hopefully it wasn't too bad. Last comment from me, I double checked OoT and this is the name they use